### DISCUSSION TITLE
Texture error

### DISCUSSION THREAD

**Discussion by iDimce** [FakeFishGames/Barotrauma#13794] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The junction box will turn black when power is applied
![image](https://github.com/FakeFishGames/Barotrauma/assets/80782649/82a2a1f1-531e-4ede-8191-c97cd6727f9f)

### Reproduction steps

1.Open submarine editor.
2.Place a junction box from [EK | Dockyard] and connect the power.
3.Connect any power-consuming device to make the junction box work.
4.Enter test mode and observe.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

submarine editor

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! This seems to be an issue with [EK | Dockyard], you might want to report the issue to the developer of the mod instead.
